kmesh_compile.sh

```diff
@@ -19,6 +19,8 @@ function run_docker_container() {
         -v /sys/fs/bpf:/sys/fs/bpf \
         -v /lib/modules:/lib/modules \
         -v "$(pwd)":/kmesh \
+        -v "$(go env GOCACHE)":/root/.cache/go-build \
+        -v "$(go env GOMODCACHE)":/root/go/pkg/mod \
```

Comment:
Yes. Because my GOPATH='/home/workspace/library'.
GOMODCACHE=$GOPATH/pkg/mod

Reply:
Also if it's not a root install of go, then GOCACHE won't be in the root folder either

Reply:
I think it's not a problem, because it is just map your GOMODCACHE of ur machine to root folder of kmesh-build docker. When kmesh-build docker next time build kmesh, it will utilize go build cache or go pkg cache of ur machine to reduce build time.

Reply:
Yes, you are right. 
I misremembered the order of mounting a folder in docker.